tools/automator-main.sh

```diff
@@ -280,7 +280,7 @@ commit() {
 
   local src_branch="${AUTOMATOR_SRC_BRANCH:-none}"
   fork_name="$src_branch-$branch-$modifier-$(hash "$title")"
-  git -c "user.name=$user" -c "user.email=$email" commit --message "$title" --author="$user <$email>"
+  git -c "user.name=$user" -c "user.email=$email" commit --message "$title" --author="$user <$email>" || true
```

Comment:
Maybe using if <nothing to commit> would be better to avoid masking real errors?